Samples/outlook-spam-reporting/README.md

```diff
@@ -39,59 +39,145 @@ To learn about key components of this sample, see [Implement an integrated spam-
+>
+> - To learn more about manifests, see [Office Add-ins manifest](https://learn.microsoft.com/office/dev/add-ins/develop/add-in-manifests).
 
-1. Download the **manifest.xml** file from this sample to a folder on your computer.
-1. Sideload the add-in manifest in Outlook on Windows (new or classic) or on the web by following the manual instructions in [Sideload Outlook add-ins for testing](https://learn.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing#sideload-manually).
-1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+### Run with the unified manifest for Microsoft 365
 
-### Run the sample from localhost
+#### Use GitHub as the web host
 
-If you prefer to host the web server for the sample on your computer, follow these steps.
+The quickest way to run the sample is to use GitHub as the web host. However, you can't debug or change the source code. The add-in web files are served from this GitHub repository.
 
-1. Install a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/) on your computer. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
-1. You need http-server to run the local web server. If you haven't installed this yet, run the following command.
+1. Download the **outlook-spam-reporting.zip** file from this sample to a folder on your computer.
+1. Sideload the sample to Outlook by following the instructions in [Sideload with the Teams Toolkit CLI (command-line interface)](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-add-in-with-unified-manifest#sideload-with-the-teams-toolkit-cli-command-line-interface).
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+
+#### Use localhost
 
-   ```console
-   npm install --global http-server
-   ```
+If you prefer to host the web server on localhost, follow these steps.
 
-1. You need Office-Addin-dev-certs to generate self-signed certificates to run the local web server. If you haven't installed this yet, you can do this with the following command.
+1. Clone or download this repository.
+1. From a command prompt, go to the root of the project folder **/samples/outlook-spam-reporting**.
+1. Run the following commands.
 
     ```console
-    npm install --global office-addin-dev-certs
+    npm install
+    npm start
     ```
 
-1. Clone or download this sample to a folder on your computer, then go to that folder in a console or terminal window.
-1. Run the following command to generate a self-signed certificate to use for the web server.
+    This starts the web server on localhost and sideloads the **manifest.json** file to Outlook.
+
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+
+    > **Note**: You can't debug a spam-reporting add-in that uses the unified manifest at this time.
```

Comment:
Just for my personal curiosity, why would this be? The JavaScript that's being debugged is the same, isn't it? Why would the type of manifest matter?

Reply:
I looked into this further and you're right; we should be able to debug since the bundle.js file of an add-in that uses the unified manifest is still added to the Wef folder. I'll correct the signature sample as well.